minijail/rust/minijail-sys/build.rs

```diff
@@ -113,6 +113,11 @@ fn main() -> io::Result<()> {
         "../../syscall_wrapper.c",
         "../../system.c",
         "../../util.c",
+        "../../libmj_perms.c",
```

Comment:
We can keep `libcap` in minijail in the first step and separate it into it's own lib later if needed. Since all the cap files are tightly coupled to minijail this is fine for now.